src/PKSim.Core/Snapshots/Mappers/SnapshotMapperBase.cs

```diff
@@ -29,7 +31,15 @@ public virtual bool IsSatisfiedBy(Type item)
 
       protected string SnapshotValueFor(string value) => !string.IsNullOrEmpty(value) ? value : null;
 
-      protected string UnitValueFor(string unit) => unit ?? "";
+      protected T? SnapshotValueFor<T>(T value, T defaultValue = default(T)) where T : struct
```

Comment:
Value will only be saved if it differs from default. Otherwise it is null. It should be used for obvious values that are almost never changed by the user (or not even accessible to the user)